### ISSUE TITLE
A11y_Model Builder_Consume_Focus Order: Focus order for the "Consume" screen is incorrect.

### ISSUE THREAD

**Issue by manaswita-chougule** [dotnet/machinelearning-modelbuilder#2682]
### GitHub Tags
#WCAG2.4.3; #A11ySev2; #A11yTCS; #A11yWCAG2.1; #DesktopApp; #Win11; #Visual Studio Client; #SH_ModelBuilder_Win32_May2023; #Focus Order; #Keyboard; #Win32;

### Environment
Application Name: Model Builder
Microsoft Visual Studio Enterprise 2022 (64-bit) - Current
Version 17.6.2
Microsoft Windows 11 Enterprise Build 22621.1702

### Repro Steps

1. Launch the Visual Studio with at least one of the .NET workloads.
2. Tab till enable ML.NET Model Builder (Tools -> Options -> Environment -> Preview Features) and select it.
3. Create a new .NET project or open an existing .NET project.
4. Open context menu (Right-click) of the project in the solution explorer.
5. Tab till Add and then tab till Machine Learning Model and press enter.
6. Model Builder dialog will open. 
7. Complete the required details of All the three(scenario, environment, data, Train, Evaluate) steps.
8. Consume screen will appear.
9. Tab till Code Snippet panel. Press tab.
10. Observe that from code snippet panel when tab is pressed focus is shifting to the feedback control and not to the next controls like Add to Solution button or Next Step button.

### Actual Result
Focus order for the "Consume" screen is incorrect. From Code snippet panel with Tab keystroke focus is shifting directly to the Feedback button skipping the previous "Add to Solution" and "Next Steps" buttons. From Feeback button with Shift+Tab keystroke focus is moving to previous controls "Add to Solution" and "Next Steps" button.

### Expected Result
Focus order for the "Consume" screen should be correct. From Code snippet panel with Tab keystroke focus should shift to the "Add to Solution" then to "Next Steps" buttons and then to Feedback button.

### User Impact
Keyboard users will not able to navigate the screen properly and face hinderance to access the controls in meaningful sequence.

### Attachment
![focus order-Consume](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/58293aab-7a41-4531-9281-de0a073acd51)

**Comment by zewditu** [comment]
@manaswita-chougule , ready to validate by Model Builder version 16.17.0.2332602.

**Comment by manaswita-chougule** [comment]
#Regressed:06-29-23; #Closed;
Verified the issue in below environment:
Visual Studio : Microsoft Visual Studio Enterprise 2022 (64-bit) - Current Version 17.6.3
Model Builder : Version 17.17.0.2332602

The issue is fixed, hence closing the bug. Please find attachment below for reference. Thanks!
![Focus order for the Consume screen is correct](https://github.com/dotnet/machinelearning-modelbuilder/assets/91586101/d6ed366c-8fc2-46d5-afe6-52db4452c5d4)
